### DISCUSSION TITLE
Found a typo

### DISCUSSION THREAD

**Discussion by Greenbean73** [ParadiseSS13/Paradise#19438] [Code Help] [answered]
There's a typo in the message for someone scratching at their skin. It says "scratches at thIEr skin" instead of "scratches at thEIr skin". Forgot where the actual bug report page was, so I put my report here
![image psd](https://user-images.githubusercontent.com/87451197/196565865-7ea9f9f1-ae0e-4c72-a7c5-5c74dd2207be.png)
![New Project](https://user-images.githubusercontent.com/87451197/196565864-d4f7c06c-b376-42e1-b500-71599c17f750.png)

**Comment by SteelSlayer** [comment, marked as answer]
Please go here and choose to make a bug report instead. https://github.com/ParadiseSS13/Paradise/issues/new/choose

**Comment by GDNgit** [comment]
Address in #19439
